benchmarks/synchrobench/src/main/java/com/yahoo/oak/synchrobench/data/eventcache/Key.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020, Verizon Media.
+ * Licensed under the terms of the Apache 2.0 license.
+ * Please see LICENSE file in the project root for terms.
+ */
+
+package com.yahoo.oak.synchrobench.data.eventcache;
+
+import com.yahoo.oak.synchrobench.contention.abstractions.BenchKey;
+
+public class Key implements BenchKey {
```

Comment:
Why Key has 2 keys inside? In general, I am lost among so many classes... Please add comment about each class what is the purpose of each.

Reply:
These are fields. I changed them to `field1` and `field2`. Also in `Value`.
I'm not sure what is the comment that you are looking for.
These are just arbitrary numbers. They have no meaning apart from resembling a real use case that we know of.